tests/bundle/SystemInfo/Registry/IdentifierBasedTest.php

```diff
@@ -17,7 +17,7 @@ class IdentifierBasedTest extends TestCase
     private IdentifierBased $registry;
 
     /**
-     * @var \PHPUnit\Framework\MockObject\MockObject[]|\Ibexa\Bundle\SystemInfo\SystemInfo\Collector\SystemInfoCollector[]
+     * @var array<string, SystemInfoCollector>
```

Comment:
I believe this shouldn't be changed and if so, then to:
`@var \PHPUnit\Framework\MockObject\MockObject[]&\Ibexa\Bundle\SystemInfo\SystemInfo\Collector\SystemInfoCollector[]